demo/README.md

```diff
@@ -0,0 +1,30 @@
+# Demo Website
+
+The demo website to show features of the measure library.
+This site fires actual gtag and measure library events, and
+displays the code for them.
+
+[To run the site, click here!](http://measurement-library.appspot.com/)
```

Comment:
I'm not sure how long we'll maintain this appengine up and running and I'm concerned that this statement might become outdated.. 
Please consider documenting only the instruction about how to run your own instance (local or a command to deploy to the app engine) and let's keep this demo side for ourselves.